### PR TITLE
opencv: bump version 4.6.0 &  4.7.0

### PR DESCRIPTION
**Pull request by granger35**
Specify library name and version:  **opencv/4.6.0 & opencv/4.7.0**

required by my projects, and also by the conan community: https://github.com/conan-io/conan-center-index/issues/15524

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by granger35** [comment]
Hi,

Now that i am an Access Request user, how could i restart the conan V1 & V2 pipelines?

Thx

**Comment by granger35** [comment]
Wait for pull request https://github.com/conan-io/conan-center-index/pull/16556 upgrading xz_utils to version 5.4.0 in ffmpeg recipe.

**Comment by danimtb** [comment]
Looks like there is a missing package in conan v1 pipeline for ffmpeg/4.4. Let's see if we can generate that package in the bracground. Menawhile, the conan v2 pipeline fails due to a conflict in the requirements: `ERROR: Version conflict: libtiff/4.4.0->libwebp/1.3.0, opencv/4.1.2->libwebp/1.2.4., that should be an easy fix just bumping the libwebp version in the opencv recipe at the moment

**Comment by danimtb** [comment]
The errors in both pipelines are coming from the ffmpeg recipe:
- On conan v2: the recipe is still not prepared to work with Conan 2.
- On conan v1: ffmpeg is failing to rebuild (I see some missing options that might be causing the missing binary). When tried to rebuild it, it fails on gcc: https://c3i.jfrog.io/c3i/misc/logs/prod/TapaholesRepo/311-linux-gcc/ffmpeg/4.4//0885adea2acd88fb7e79b4bda9709c5ac37386ff-build.txt

**Comment by granger35** [comment]
Thx @danimtb for pointing me the ffmpeg logs showing the build issue related to vaapi which was not found, while libva seems to be installed with apt-get in the log. Strange behavior!

File: recipes/opencv/4.x/conanfile.py [hunk 1]
```diff
@@ -200,14 +200,17 @@ def requirements(self):
         if self.options.with_eigen:
             self.requires("eigen/3.3.9")
```
Comment (czoido) [inline recipes/opencv/4.x/conanfile.py:201]: Could we also bump Eigen required version to 3.4.0?
```suggestion
            self.requires("eigen/3.4.0")
```

**Comment by guillaume-michel** [comment]
Hello!
What is the status of this PR please?

**Comment by innerlee** [comment]
Bump. 4.8.0 was released also

**Comment by ilya-lavrenov** [comment]
Can be closed in favor of https://github.com/conan-io/conan-center-index/pull/19194

**Comment by uilianries** [comment]
@granger35 Thank you for your contribution! Similar changes have been integrated into CCI by the PR #19194 and now this PR will be closed.

**Pull request closed without merging**